tests/OrderableRowsThroughVersionedTest.php

```diff
@@ -33,7 +33,7 @@ class OrderableRowsThroughVersionedTest extends SapphireTest
     protected function setUp(): void
     {
         parent::setUp();
-        $this->orignalReadingMode = Versioned::get_reading_mode();
+        $this->originalReadingMode = Versioned::get_reading_mode();
```

Comment:
See line 31 above - this was a typo which resulted in a PHP deprecation warning in CI. Super minor, doesn't really matter until they actually remove the ability to have dynamic properties and it's only a unit test so I don't think it's worth backporting.